lib/pac55xx/can.c

```diff
@@ -0,0 +1,470 @@
+/**
+ * @addtogroup can_api CAN Peripheral API
+ * @ingroup peripheral_apis
+ * @brief <b>PAC55xxxx CAN Driver</b>
+ * @author @htmlonly &copy; @endhtmlonly 2020 Kevin Stefanik <kevin@allocor.tech>
+ * @date February 13, 2020
+ *
+ * This library supports the CAN module in the PAC55xx SoC from Qorvo.
+ *
+ * Note: Acceptance Code Mask Register values of 1 indicate the filter is to
+ * ignore the bit. However, standard CAN driver APIs use a positive logic for the
+ * mask. The implementations in this file inverts masks as appropriate to
+ * the mask to make this more portable/intuitive.
```

Comment:
I think somewhere in this header intro is where you should document the reasoning behind the bswap32 stuff please, otherwise, it's only tracked in this github pr as far as I can tell.  It's a fairly visible WTF at the moment, even though we've got good reasons for it now :)